drivers/pinmux/stm32/pinmux_stm32l4x.h

```diff
@@ -101,13 +105,17 @@
 #define STM32L4X_PINMUX_FUNC_PB3_SPI3_SCK                                      \
 	(STM32_PINMUX_ALT_FUNC_6 | STM32_PUSHPULL_NOPULL)
 
+#define STM32L4X_PINMUX_FUNC_PB4_I2C3_SDA                                      \
```

Comment:
Specific to STM32L496 soc variant. Should be under #ifdef

Reply:
Done.